consul_exporter.go

```diff
@@ -93,6 +93,11 @@ var (
 		"Status of health checks associated with a service.",
 		[]string{"check", "node", "service_id", "service_name", "status"}, nil,
 	)
+	serviceCheckNames = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_check_name"),
+		"Link the service id and check name if available",
+		[]string{"service_id", "check_name"}, nil,
```

Comment:
Since the metric tracks the service:checks mapping, the name should be `consul_service_checks` and the labels should be `service_id`, `service_name`, `check_id` and `check_name`.

Reply:
Updated PR.